Makefile

```diff
@@ -46,6 +47,10 @@ all: hypervisor devicemodel tools
 hypervisor:
 	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT) BOARD=$(BOARD) FIRMWARE=$(FIRMWARE) RELEASE=$(RELEASE) clean
 	make -C $(T)/hypervisor HV_OBJDIR=$(HV_OUT) BOARD=$(BOARD) FIRMWARE=$(FIRMWARE) RELEASE=$(RELEASE)
+ifeq ($(PLATFORM),uefi)
+	echo "building hypervisor as EFI executable..."
+	make -C efi-stub HV_OBJDIR=$(HV_OUT) EFI_OBJDIR=$(EFI_OUT)
```

Comment:
acrn.efi = acrn.bin + boot.efi,  then i move it out from hypervisor
T := $(CURDIR)    current directory, here it is in acrn-hypervisor folder, anyway will add $(T)

Reply:
> acrn.efi = acrn.bin + boot.efi, then i move it out from hypervisor

I see, thanks for the explanation.

> T := $(CURDIR) current directory, here it is in acrn-hypervisor folder, anyway will add $(T)

I agree, I don't really know (remember) why we have this $(T) everywhere but since we do, we might as well be consistent.